src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -93,59 +93,166 @@
         </v-row>
         <v-row>
           <v-col>
-            <h3>Is this program or course recognized by the ECE Registry?</h3>
-            <p class="mt-3">
-              To be certified in B.C. as an Early Childhood Educator (ECE) requires the successful completion of an early childhood education training program
-              that's recognized by the ECE Registry, or a program that's deemed equivalent. ECE Assistant certification requires successful completion of a
-              course that's a part of a recognized early childhood education training program or deemed equivalent.
-            </p>
-            <p class="mt-3">
-              Check the list of
-              <a
-                href="https://www2.gov.bc.ca/gov/content/education-training/early-learning/teach/training-and-professional-development/become-an-early-childhood-educator/recognized-ece-institutions"
-              >
-                recognized programs
-              </a>
-              if you're not sure if this program is recognized
-            </p>
+            <h3>Where did you take it?</h3>
           </v-col>
         </v-row>
         <v-row>
-          <v-col>
-            <p>Is this program on the recognized list?</p>
-            <v-radio-group v-model="educationRecognition" :rules="[Rules.requiredRadio('Select an option')]">
-              <v-radio v-for="(recognition, index) in educationRecognitionRadio" :key="index" :label="recognition.label" :value="recognition.value"></v-radio>
-            </v-radio-group>
+          <v-col md="8" lg="6" xl="4">
+            Country
+            <v-select
+              class="pt-2"
+              :items="configStore.countryList"
+              variant="outlined"
+              label=""
+              v-model="country"
+              item-title="countryName"
+              item-value="countryId"
+              :rules="[Rules.required('Select your country', 'countryId')]"
```

Comment:
we don't need countryId here, Rules.required only needs 1 argument. 

const required = (message = "This field is required")